Vagrantfile

```diff
@@ -1,86 +1,46 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
-VAGRANTFILE_API_VERSION = '2'.freeze
-
-Vagrant.require_version '>= 1.5.0'
+def plugin_installed?(name)
+  unless Vagrant.has_plugin?(name)
```

Comment:
Not sure if I follow here because I'm not an versed Rubyist, but:

```
def plugin_installed?(name)
  return unless Vagrant.has_plugin?(name)
  puts "Vagrant plugin '#{name}' is NOT installed."
  puts 'Please run:'
  puts "\tvagrant plugin install #{name}"
  exit(1)
end
```

```
➜  chef-stackstorm git:(update-vagrantfile) ✗ vagrant plugin list
vagrant-omnibus (1.5.0)
vagrant-share (1.1.5, system)
➜  chef-stackstorm git:(update-vagrantfile) ✗
```

doesn't work:

```
➜  chef-stackstorm git:(update-vagrantfile) ✗ vagrant status
Vagrant plugin 'vagrant-omnibus' is NOT installed.
Please run:
       	vagrant plugin install vagrant-omnibus
Vagrant plugin 'vagrant-omnibus' is NOT installed.
Please run:
       	vagrant plugin install vagrant-omnibus
➜  chef-stackstorm git:(update-vagrantfile) ✗
```